docs/rules/lines-between-class-members.md

```diff
@@ -0,0 +1,152 @@
+# enforce lines between class members (lines-between-class-members)
+
+This rule is to improve readability by enforce lines between class methods.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```js
+/* eslint lines-between-class-members: ["error", "always"]*/
+class MyClass {
+  foo() {
+    //...
+  }
+  bar() {
+    //...
+  }
+}
+```
+
+Examples of **correct** code for this rule:
+
+```js
+/* eslint lines-between-class-members: ["error", "always"]*/
+class MyClass {
+  foo() {
+    //...
+  }
+
+  bar() {
+    //...
+  }
+}
+```
+
+### Options
+
+This rule has one option, which can be a string option or an object option.
+
+String option:
+
+* `"always"`(default) requires line breaks between methods
+* `"never"` disallows line breaks between methods
+
+Object option:
+
+* `"multiline": always` requires line breaks if the method is multiline
```

Comment:
Can these be formatted to either: `"multiline": "always"` or `multiline: "always"`. The current formatting doesn't match JSON or object syntax.